packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -34,12 +35,21 @@ const prepareHeaders = async (headers: Headers) => {
 export const orchestratorApi = createApi({
     reducerPath: 'orchestratorApi',
     baseQuery: (args, api, extraOptions: ExtraOptions) => {
-        const { baseQueryType } = extraOptions || {};
+        const { baseQueryType, customBaseQueryType } = extraOptions || {};
 
         const state = api.getState() as RootState;
         const { orchestratorApiBaseUrl, graphqlEndpointCore } =
             state.orchestratorConfig;
 
+        const customBaseQueries = state.customBaseQueries;
+
+        if (customBaseQueries) {
+            for (const query of customBaseQueries) {
+                if (query.queryType === customBaseQueryType) {
```

Comment:
Maybe we should add a check and an error message for the case when there is a customBaseQuery and a queryType but the queryType doesn't match any of the customBaseQueris and none of the BaseQueryTypes. Currently that means it will catch the default clause and run it as a graphQlQuery with the default url. 

It might be useful to have an error when you misttype or forget to implement something here